build.gradle.kts

```diff
@@ -60,7 +60,7 @@ val activationVersion = "2.1.3"
 val jaxbVersion = "4.0.5"
 val jaxbApiVersion = "4.0.2"
 val openApiToolsVersion = "0.2.6"
-
+val temporalVersion = "1.26.1"
```

Comment:
why not the 1.27.0?

Reply:
You are right:  the 1.27.0 version is the last one

Reply:
let's change the dependency to 'io.temporal:temporal-sdk:1.27.0'